docs/app/pages/components/components-sections/notifications/badge/badge.component.html

```diff
@@ -0,0 +1,157 @@
+<div class="badge-separator">
+    <button class="btn btn-primary" [uxBadge]="badgeContent" [badgeSize]="badgeSize" [badgeMaxValue]="badgeMaxValue"
+        [badgeOverlap]="badgeOverlap" [badgeColor]="selectedColor.name" [badgeHidden]="badgeHidden"
+        [badgeHorizontalPosition]="badgeHorizontalPosition" [badgeVerticalPosition]="badgeVerticalPosition">
+        Messages
+    </button>
+    <ux-icon class="icon-example" name="filter" [uxBadge]="badgeContent" [badgeSize]="badgeSize"
+        [badgeMaxValue]="badgeMaxValue" [badgeOverlap]="badgeOverlap" [badgeColor]="selectedColor.name"
+        [badgeHidden]="badgeHidden" [badgeHorizontalPosition]="badgeHorizontalPosition"
+        [badgeVerticalPosition]="badgeVerticalPosition">
+        Messages
+    </ux-icon>
+</div>
+
+<hr />
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <ux-accordion>
+            <ux-accordion-panel class="accordion-chevron" heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-6 col-sm-12">
+                        <label for="uxBadge">uxBadge</label>
+                        <div class="input-group m-b-sm">
+                            <input type="text" placeholder="Enter description" class="form-control"
+                                [(ngModel)]="badgeContent">
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-12">
+                        <label for="badgeMaxValue">badgeMaxValue</label>
+                        <ux-number-picker [(value)]="badgeMaxValue" min="0" max="1000000"></ux-number-picker>
```

Comment:
Could you use `ux-number-picker-inline` in the customize section? It will look much better in the Micro Focus documentation site.